projects/telemetry/config.json

```diff
@@ -22,113 +21,20 @@
-                        "loop": true,
-                        "volume": 0
-                    }
-                },
-
-                "view":
-                {
-                    "type": "flat",
-                    "options":
-                    {
-                        "repeatX": false,
-                        "repeatY": false
-                    }
-                },
-
-                "plugins":
-                {
-                    "configurations":
-                    [
-                        {
-                            "instance": "accelerometer",
-
-                            "data":
-                            {
-                                "json": "https://cdn.forgejs.org/samples/metadata/GOPR7391_gpmf_accl.json"
-                            },
-
-                            "options":
-                            {
-                                "dom": true
-                            }
-                        },
-
-                        {
-                            "instance": "altimeter",
-
-                            "data":
-                            {
-                                "json": "https://cdn.forgejs.org/samples/metadata/GOPR7391_gpmf_altitude.json"
-                            },
-
-                            "options":
-                            {
-                                "dom": true
-                            }
-                        },
-
-                        {
-                            "instance": "compass",
-
-                            "data":
-                            {
-                                "json": "https://cdn.forgejs.org/samples/metadata/GOPR7391_gpmf_gyro.json"
-                            },
-
-                            "options":
-                            {
-                                "dom": true
-                            }
-                        },
-
-                        {
-                            "instance": "speedometer",
-
-                            "data":
-                            {
-                                "json": "https://cdn.forgejs.org/samples/metadata/GOPR7391_gpmf_speed.json"
-                            },
-
-                            "options":
-                            {
-                                "dom": true
-                            }
-                        }
-                    ]
-                }
-            },
-
-            {
-                "uid": "scene-1",
-                "name": "Second scene",
-                "slug": "second-scene",
-                "description": "This is the second scene",
-
-                "media":
-                {
-                    "uid": "media-1",
-                    "type": "video",
-
-                    "source":
-                    {
                         "format": "equi",
-                        "url": "https://cdn.forgejs.org/samples/metadata/GOPR7391.mp4"
+                        "levels": [
```

Comment:
Levels are not working because of this line, needs some changes https://github.com/gopro/forgejs/blob/master/src/render/RenderManager.js#L750

Reply:
Fixed with https://github.com/gopro/forgejs/pull/150

Reply:
Fixed